app/decorators/ambassador_task_assignment_decorator.rb

```diff
@@ -0,0 +1,18 @@
+class AmbassadorTaskAssignmentDecorator < ApplicationDecorator
+  delegate_all
+  alias ambassador_task_assignment object
+
+  def button_to_toggle_completion_status(current_user, current_organization)
+    return unless current_user.ambassador?
+
+    is_complete = ambassador_task_assignment.completed?
+    button_label = is_complete ? "Mark Pending" : "Mark Complete"
+
+    h.button_to(
+      button_label,
+      h.organization_ambassador_task_assignment_url(current_organization, ambassador_task_assignment),
+      method: :put,
```

Comment:
Would you add a bootstrap button class to this too?

Reply:
Good catch, thanks! Updated

<img width="487" alt="Screen Shot 2019-06-03 at 3 16 59 PM" src="https://user-images.githubusercontent.com/4433943/58828157-cf7b2c00-8612-11e9-85aa-a026228cd523.png">